waku.md

```diff
@@ -209,17 +209,29 @@ It is only relevant if you want to decrypt the incoming message, but if you only
 
 Data field contains encrypted message of the Envelope. In case of symmetric encryption, it also contains appended Salt (a.k.a. AES Nonce, 12 bytes). Plaintext (unencrypted) payload consists of the following concatenated fields: flags, auxiliary field, payload, padding and signature (in this sequence).
 
-	flags: 1 byte; first two bits contain the size of auxiliary field, third bit indicates whether the signature is present.
-	
-	auxiliary field: up to 4 bytes; contains the size of payload.
+Using [Augmented Backus-Naur form (ABNF)](https://tools.ietf.org/html/rfc5234) we have the following format:
+
+```
+bytes           = [repeat] OCTET
```

Comment:
repeat is not a valid rule in ABNF. This should probably be `*OCTET` (or maybe 1*OCTET)

Can try here for validation https://oskarth.github.io/panini/
![image](https://user-images.githubusercontent.com/1552237/69601614-b1393a80-104f-11ea-8ebb-47c377f61657.png)